src/Assignment-2/ContactManager.cs

```diff
@@ -0,0 +1,51 @@
+﻿// <copyright file="ContactManager.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+namespace Assignment_2
+{
+    /// <summary>
+    /// Contact class will manage the contact details of the a particular person.
+    /// </summary>
+    public class ContactManager
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ContactManager"/> class.
+        /// </summary>
+        /// <param name="name">Name of the person</param>
+        /// <param name="phoneNumber">Phone number of the person</param>
+        /// <param name="email">Email ID of the person</param>
+        /// <param name="note">Description of the person</param>
+        public ContactManager(string name, string phoneNumber, string email, string note)
+        {
+            this.Name = name;
+            this.PhoneNumber = phoneNumber;
+            this.Email = email;
+            this.Note = note;
+        }
+
+        /// <summary>
+        /// Gets or Sets the Name of the person
```

Comment:
Please make the description more crisp.
```suggestion
        /// Name of the person
```
Do change in all the occurrences also.

Reply:
Actually it was the template bro, if I try to remove the "Gets or Sets" it shows warning to add "Gets or Sets" in the beginning of the line.